exercises/practice/space-age/.approaches/where-clause/content.md

```diff
@@ -0,0 +1,22 @@
+# Give meaningful names to important values
+
+```haskell
+ageOn :: Planet -> Float -> Float
+ageOn planet seconds = seconds / (periodInEarthYears * secondsPerEarthYear)
+  where
+    secondsPerEarthYear = 60 * 60 * 24 * 365.25
+
+    periodInEarthYears = case planet of
+      Mercury -> 0.2408467
+      Venus -> 0.61519726
+      Earth -> 1
+      Mars -> 1.8808158
+      Jupiter -> 11.862615
+      Saturn -> 29.447498
+      Uranus -> 84.016846
+      Neptune -> 164.79132
+```
+
+This approach uses a `case` expression to choose the relevant orbital period.
+Also a `where` clause is used to give names to important values.
+This tends to greatly improve readability.
```

Comment:
I feel like this content is what should be in the `introduction.md` document, and what's there should be in this content.
The `introduction.md` document should be relatively succinct, whereas the approach can do into a lot more detail.

Reply:
I'm unhappy with this file too.

Reasons I did it this way:

- For most (all?) other exercises, `introduction.md`'s current content really is _general guidance_. (In fact, the bit about `where` is copied verbatim from another approaches introduction. The bit about `case` might well be reused as well.)
- If/when another approach comes along, it _very_ likely uses `case` and `where` as well.

If it were up to me, `content.md` and `snippet.txt` would be left out until another approach is added, but I interpreted the docs as saying that they are required.

Reply:
> If it were up to me, content.md and snippet.txt would be left out until another approach is added, but I interpreted the docs as saying that they are required.

I'm fine with that.

Reply:
I have removed them.